From: Jan Engelhardt <jengelh@inai.de>
Date: pam_ccreds-db6.diff

src: fix sloppy DB version >= 5 check

db-6.0 is on the scene.

---
 cc_db.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: pam_ccreds-10/cc_db.c
===================================================================
--- pam_ccreds-10.orig/cc_db.c
+++ pam_ccreds-10/cc_db.c
@@ -66,7 +66,7 @@ int pam_cc_db_open(const char *filename,
 		return PAM_SERVICE_ERR;
 	}
 
-#if (DB_VERSION_MAJOR > 3) && (DB_VERSION_MINOR > 0)
+#if DB_VERSION_MAJOR > 4 || (DB_VERSION_MAJOR == 4 && DB_VERSION_MINOR >= 1)
 	rc = db->open(db, NULL, filename, NULL,
 		      DB_BTREE, db_flags, mode);
 #else
